CHANGELOG.md

```diff
@@ -7,211 +7,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Revert "Removing recovery video entry from changelog"
-- Merge remote-tracking branch 'origin/develop' into sync-master
-- Merge remote-tracking branch 'origin/develop' into sync-master
-- Merge remote-tracking branch 'origin/develop' into sync-master
-- Merge remote-tracking branch 'origin/develop' into sync-master
+### Added
+- [#10905](https://github.com/MetaMask/metamask-extension/pull/10905): Implemented basic switchEthereumChain feature
+- [#10967](https://github.com/MetaMask/metamask-extension/pull/10967): Add setting to dismiss seed phrase backup reminder
+- [#10293](https://github.com/MetaMask/metamask-extension/pull/10293): Implement Ledger Live bridge
+- [#10962](https://github.com/MetaMask/metamask-extension/pull/10962): Add backwards compatibility for ETH <-> WETH contract address validation
+- [#11131](https://github.com/MetaMask/metamask-extension/pull/11131): What's New: Adding notification for updated seed phrase wording
+- [#11083](https://github.com/MetaMask/metamask-extension/pull/11083): Adding BSC Swaps notification when if a user is on BSC Mainnet
+- [#11088](https://github.com/MetaMask/metamask-extension/pull/11088): Swaps: Add specific error content if Contract data are not enabled on Ledger
+- [#11064](https://github.com/MetaMask/metamask-extension/pull/11064): Supporting EIP-712 signing for Ledger accounts
 
-## [9.5.3]
-### Uncategorized
-- [#11131](https://github.com/MetaMask/metamask-extension/pull/11131): Adding notification for updated seed phrase wording
-- [#11130](https://github.com/MetaMask/metamask-extension/pull/11130): Sync `master` with `develop`
-- Bumping package.json
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- [#11127](https://github.com/MetaMask/metamask-extension/pull/11127): Fix a condition for checking if a token should be added
+### Changed
+- [#11007](https://github.com/MetaMask/metamask-extension/pull/11007): Hide basic tab in advanced gas modal when on testnets.
+- [#10994](https://github.com/MetaMask/metamask-extension/pull/10994): Replacing the text "Seed Phrase" with "Secret Recovery Phrase" throughout the application.
+- [#10987](https://github.com/MetaMask/metamask-extension/pull/10987): Swaps: Improve hardware wallet UX
+- [#10936](https://github.com/MetaMask/metamask-extension/pull/10936): Swaps: Allow 0% slippage, show a warning for 0 < slippage <= 1, disallows Reviews when negative slippage occurs.
+- [#10946](https://github.com/MetaMask/metamask-extension/pull/10946): Send user to activity view after transaction is complete
+- [#10767](https://github.com/MetaMask/metamask-extension/pull/10767): Improving error handling when gas prices are unavailable.
 - [#11118](https://github.com/MetaMask/metamask-extension/pull/11118): Removing support survey notification from What's New
-- [#10956](https://github.com/MetaMask/metamask-extension/pull/10956): Handling custom token decimal fetch failure due to network error
```

Comment:
I believe that 10956 should be included in this changelog

Reply:
I reworded and included it here: https://github.com/MetaMask/metamask-extension/pull/11133/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR43